_posts/2019-10-18-Reply-to-Childhood-Vision.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: Vision Of Your Future Self
+date: 2019-10-18 20:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [Me]
+image:
+image2:
+author: Valentino Urbano
+---
+
+[Reply to Childhood Vision][1]:
+
+> A clear mental picture appears to me when I try to imagine who I want to be
+
+This sounds like a great writing prompt for a "What to write about" column.
+
+What would YOU image?
+
+[Started as a comment, but was getting too long]
+
+My idea is less abstract, and more material I guess. I share the idea of this scholar aurea though.
+
+I have already laid out my goals for the next 5 years:
+
+- [The Bigger Picture][2]
+
+The first item of which I've been trying to reach for a while, but with the current price of housing has taken a long time (and it is still going to take a while):
+
+- A house in the countryside with lots of green.
+
+On this front I hopefully will be able to share something more about it sometime next year.
+
+The other points in there are easier.
+
+Apart the happiness one.
+
+I feel that as humans we're always envy, always looking at others. It is not good nor healthy for us. You should never compare yourself to others. You are different so you're no lower or higher than other people, you are just you.
+
+Focus on what makes you happy and what you wanna do and who you wanna be.
+
+Back to the house:
+
+I guess I love what it represents. The freedom of living and working surrounded by nature after living in big cities my whole life I have enough.
```

Comment:
"I have enough" seems disjointed from the sentence.

Reply:
Change it entirely. Not sure about the end of the sentence though